### PR TITLE
Adding rules.md for infra folders and registry keys

### PR DESCRIPTION
**Pull request by AndreiCsibi-msft**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/test-base/rules.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ms.service: virtual-desktop
+ms.localizationpriority: medium
+ms.collection: TestBase-M365
+ms.custom:                 # This is an optional, free-form field you can use to define your own collection of articles. If you have more than one value, format as a bulleted list. This field truncates to something like 144 characters (inclusive of spaces) so keep it short.
+ms.reviewer: mapatel
+f1.keywords: NOCSH
+---
+# Application / Test rules
+
+All application or tests in Test Base need to follow the following rules:
```
Comment (JohanFreelancer9) [inline microsoft-365/test-base/rules.md]:
```suggestion
All applications or tests in Test Base need to comply with the following rules:
```

File: microsoft-365/test-base/rules.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Test Base Folders
```
Comment (JohanFreelancer9) [inline microsoft-365/test-base/rules.md]:
```suggestion
## Test Base folders 
```

File: microsoft-365/test-base/rules.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+* %SYSTEMDRIVE%\ProcMon
+* %SYSTEMDRIVE%\PSTools
+* %SYSTEMDRIVE%\TokenProviderTool
+* %SYSTEMDRIVE%\USLPowershellModules
+* %SYSTEMDRIVE%\UtcUtil
+* %SYSTEMDRIVE%\WPT
+* %SYSTEMDRIVE%\WULogs
+
+> [!IMPORTANT]
+> **Avoid the following:**
```
Comment (JohanFreelancer9) [inline microsoft-365/test-base/rules.md]:
```suggestion
> **Avoid the following**:
```

File: microsoft-365/test-base/rules.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+> * Blocking the execution of any process from these folders. If your application is an anti-malware software, configure your app installation to allow unimpeded execution of all processes from these folders.
```
Comment (JohanFreelancer9) [inline microsoft-365/test-base/rules.md]:
```suggestion
> * Blocking the execution of any process from these folders. If your application is anti-malware software, configure your app installation to allow unimpeded execution of all processes from these folders.
```

File: microsoft-365/test-base/rules.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+> * Tampering with of these folders.
```
Comment (JohanFreelancer9) [inline microsoft-365/test-base/rules.md]:
```suggestion
> * Tampering with any of these folders.
```

File: microsoft-365/test-base/rules.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+## Test Base Registry Keys
```
Comment (JohanFreelancer9) [inline microsoft-365/test-base/rules.md]:
```suggestion
## Test Base registry keys
```

File: microsoft-365/test-base/rules.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+The applications/tests should not delete/modify any registry keys related to:
```
Comment (JohanFreelancer9) [inline microsoft-365/test-base/rules.md]:
```suggestion
The applications/tests should not delete or modify any registry keys related to:
```

File: microsoft-365/test-base/rules.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+* Windows Telemetry level
```
Comment (JohanFreelancer9) [inline microsoft-365/test-base/rules.md]:
```suggestion
* Windows telemetry level
```

File: microsoft-365/test-base/rules.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+ms.topic: how-to
+ms.date: 07/06/2021
+ms.service: virtual-desktop
+ms.localizationpriority: medium
+ms.collection: TestBase-M365
+ms.custom:                 # This is an optional, free-form field you can use to define your own collection of articles. If you have more than one value, format as a bulleted list. This field truncates to something like 144 characters (inclusive of spaces) so keep it short.
+ms.reviewer: mapatel
+f1.keywords: NOCSH
+---
+# Application / Test rules
```
Comment (JohanFreelancer9) [inline microsoft-365/test-base/rules.md]:
```suggestion
# Application/Test rules
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @AndreiCsibi-msft, kindly commit the suggested changes if you agree with them. Many thanks.

**Comment by yogkumgit** [comment]
@AndreiCsibi-msft Gentle remnder

**Comment by AndreiCsibi-msft** [comment]
> @AndreiCsibi-msft Gentle remnder

@JohanFreelancer9, @yogkumgit : I'll take a look at these comments later today. Thanks a lot!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mansipatel-usl Please merge this PR. Thanks

**Comment by AndreiCsibi-msft** [comment]
@yogkumgit: Is there anyone else who can review/merge this PR if needed? Mansi is OOF for another week.

**Comment by yogkumgit** [comment]
@tiburd Please help

**Comment by tiburd** [comment]
@AndreiCsibi-msft 

This pull request was opened in the public repo. PMs and content developers should work in the private repo, per the [MARVEL Authoring Guide for Docs](https://review.docs.microsoft.com/en-us/office-authoring-guide/office-docs-authoring-overview?branch=master). 

Please create a PR in the private repo to submit this content update. We'll close this PR. 

Thanks!

**Pull request closed without merging**